widgetssdk/src/main/java/com/glia/widgets/di/Dependencies.java

```diff
@@ -58,6 +60,10 @@ public static void onAppCreate(Application application) {
                 new ApplicationLifecycleManager(),
                 controllerFactory.getChatHeadController()
         );
+        ActivityWatcher activityWatcher = new ActivityWatcher(
+                application,
+                controllerFactory.getCallVisualizerController());
+        activityWatcher.init();
```

Comment:
can remove this step

Reply:
Yes, indeed, it's possible to remove `activityWatcher.init();` after changing `fun init() {}` to init `{}`.